downstream.go

```diff
@@ -3398,6 +3674,8 @@ func forwardChannel(ctx context.Context, dc *downstreamConn, ch *upstreamChannel
 	if !dc.caps.IsEnabled("soju.im/no-implicit-names") && !dc.caps.IsEnabled("draft/no-implicit-names") {
 		sendNames(ctx, dc, ch)
 	}
+
+	dc.sendChannelMetadata(ctx, ch.Name)
```

Comment:
This handles channel metadata, but we're missing support for users. We need to check for monitored users and send updates accordingly if `draft/metadata-notify-2` is enabled.

Reply:
Will be done in a separate MR